### DISCUSSION TITLE
AWS stack creation error with S3 (Bucket cannot have ACLs set with ObjectOwnership's BucketOwnerEnforced setting)

### DISCUSSION THREAD

**Discussion by sarurururu** [mozilla/hubs#6061] [Hubs Cloud Questions]
**Description**

Error with the 1.1.5 AWS cloudformation stack creation:

_Bucket cannot have ACLs set with ObjectOwnership's BucketOwnerEnforced setting_

**Comment by Frans** [comment]
Same here created a new stack, got this error:
`Bucket cannot have ACLs set with ObjectOwnership's BucketOwnerEnforced setting (Service: Amazon S3; Status Code: 400; Error Code: InvalidBucketAclWithObjectOwnership; Request ID: QXW1TEDCSHMZQGPY; S3 Extended Request ID: oFX6HtOyaBNJMlMtCBZqdXWqx/SfG/347zd2GLfbiUKzCnp8yDssPB8Rp+M/IZ3vC3+WZrPZB79L5iEEJCJ5sN9DJzriCZElCt2TGgGAYmM=; Proxy: null)`

**Comment by pattersonbl2** [comment]
There is a workaround pinned at the top for this issue.

**Reply by thiagopsnfg** [reply]
@pattersonbl2 where? i have the same issue here.

**Reply by vikhyat187** [reply]
@thiagopsnfg @pattersonbl2 can you please point me to the link, didn't find it.
